client/src/components/user/Password.js

```diff
@@ -0,0 +1,98 @@
+import React, { useEffect, useState } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import Container from "@material-ui/core/Container";
+import TextField from "@material-ui/core/TextField";
+import Button from "@material-ui/core/Button";
+import { makeStyles } from "@material-ui/core/styles";
+import { saveNewPassword } from "../../store/user/actions";
+
+export default function Password({ closeModal, userId }){
+
+    const dispatch = useDispatch();
+    const classes = useStyles();
+    const [newPassword, setNewPassword] = useState();
+    const [confirmedPassword, setConfirmedPassword] = useState();
+
+    return (
+        <Container classes={{
+            root: classes.root
+          }}>
+            <TextField
+                label = "New password"
+                className={classes.textField}
+                margin="normal"
+                onChange={({ currentTarget }) => {
+                    setNewPassword(currentTarget.value);
+                }
+             } 
+            />
+            <TextField
+                label = "Confirm password"
+                className={classes.textField}
+                margin="normal"
+                onChange={({ currentTarget }) => {
+                    setConfirmedPassword(currentTarget.value);
+                }
+             } 
+            />
+
+    <Button
+        variant="contained"
+        color="primary"
+        className={classes.button}
+        onClick={() => {
+          if(confirmedPassword === newPassword){
+              dispatch(
+                  saveNewPassword({
+                    userId,   
+                    password: newPassword,
+                    callback: () => {
+                        closeModal();
+                    }
+                })
+              );
+          }else{
+              console.log("They are not equal");
+            }          
+        }}
```

Comment:
ovo `console.log("Ther are not equal")` nista ne znaci, korisnik nece viditi nista. Stavi umjesto toga alert('Poruka koju hoces da prikazes')

Reply:
Znam, znam. To sam zbog sebe nesto..ko šiša korisnika haha